### DISCUSSION TITLE
Datei-Backup nur 3 von 4 Ordner?

### DISCUSSION THREAD

**Discussion by scannergmbh** [FriendsOfREDAXO/.github#20] [Q&A]
ich hab mir mal die Demo drauf installiert. Wenn ich "Daten"-Backup über das Addon mache. Wiso wird der Ordner "redaxo" nicht angeboten? Es werden nur media, assets und recources gesichert.
Ich habe einen Umzug auf einen zweiten Testserver simuliert. Da fehlt ja dann letztendlich der "redaxo"-Ordner.
Wie geht man dann richtig vor, wenn man ein Produkt von einem Testserver auf einen Liveserver stellen möchte?

UPDATE: gerade erst gesehen. Denke mal hier finde ich die Antwort ...
[https://redaxo.org/doku/main/umzug]

... unter dem oben stehenden Link wird der zweite Punkt beschrieben: "Download aller Ordner und Dateien der REDAXO-Installation, z. B. per SFTP (Transfer-Einstellungen des FTP-Programms auf binary/binär einstellen)"

Wenn ich das richtig sehe, werden beim Serverumzug manuell über FTP die Daten gesichert.
Was für einen Sinn macht dann das Backup-Addon, wenn nur drei von vier Ordner gesichert werden?

**Comment by eaCe** [comment]
Der Sinn hinter dem Backup-Addon ist es Dateien zu sichern, die durch Redakteur*innen angelegt wurden oder spezielle Konfigurationen von Addons (im Assets-Ordner). Oder eben auch eigene Styles und Scripts. Wenn REDAXO auf einem neuen Server installiert wird, kann man die Datenbank und z.B. den Media Ordner kopieren, die gewünschten Addons installieren und im besten Falle sollte alles problemlos laufen. Das Backup-Addon ist kein exportiere alle Ordner-Addon. Hat man viele Addons im Einsatz, die eventuell noch Vendoren einschließen, kann es auch schnell zu Timeouts kommen.

Weitere Informationen gibt es hier: https://www.redaxo.org/doku/main/backup

Gerne auch einfach in Slack nachfragen.
